llvm/test/Analysis/DXILResource/buffer-frombinding.ll

```diff
@@ -46,14 +46,14 @@ define void @test_typedbuffer() {
   ; Buffer<uint4> Buf[24] : register(t3, space5)
   %typed2 = call target("dx.TypedBuffer", <4 x i32>, 0, 0, 0)
       @llvm.dx.handle.fromBinding.tdx.TypedBuffer_i32_0_0t(
-          i32 2, i32 7, i32 24, i32 0, i1 false)
+          i32 5, i32 3, i32 24, i32 0, i1 false)
```

Comment:
What happened here?

Reply:
That was supposed to be included in #100699 in response to [a comment](https://github.com/llvm/llvm-project/pull/100699#discussion_r1715835841), but I guess I somehow put it in the wrong change. Committing that separately here: #105409